modules/widgets/twitter-timeline.php

```diff
@@ -386,6 +390,11 @@ class="widefat"
 				value="<?php echo esc_attr( $instance['tweet-limit'] ); ?>"
 			/>
 		</p>
+		<p>
```

Comment:
I don't know if we really need this if the interface already greys out the option. That may be enough to indicate how this works. What do you think?

Reply:
Hmm, I'm not so sure about this. I suspect not including a notice along with graying the above input/label out will confuse some people and lead to the same sort of issue that led to the opening of this issue.

That's my opinion at least. Happy to omit this if needed :)